steward/src/commands/keys/cosmos/add.rs

```diff
@@ -6,9 +6,13 @@ use rand_core::OsRng;
 use signatory::FsKeyStore;
 use std::path;
 
+/// Gorc keys cosmos add [name]
 #[derive(Command, Debug, Default, Parser)]
+#[clap(
+    long_about = "DESCRIPTION \n\n Create a new Cosmos Key.\n This command creates a new Cosmos key with a name as a String.\n It has an overwrite option, a boolean which can be true or false."
+)]
 pub struct AddCosmosKeyCmd {
-    pub args: Vec<String>,
```

Comment:
I think args might be the more apt identifier if we expect to have to process any other arguments, otherwise if we're just planning on processing the name perhaps this is better as a stand alone string instead of a vector?

Reply:
Hey Phil, I think it'll be best to use individually process the arguments instead of using a vector. This because using a vector will render this:

```
ARGS:
    <ARGS>...  
```
However, for commands with multiple args like this:
https://github.com/PeggyJV/steward/blob/272a5a294af62b5a2398d9bec656938482a96bee/steward/src/commands/cosmos_to_eth.rs#L46-L53

The user won't be able to know what arguments are needed since it'll still display:

```
ARGS:
    <ARGS>...  
```

Reply:
Agreed, individually processing the arguments sounds sensible.